extensions/src/components/extensions/index.js

```diff
@@ -2,14 +2,20 @@
 import createExtension from '@/tools/component-wrapper';
 
 import testEntityComponents from './test';
+import meModelComponents from './me-model';
+import eModelComponents from './e-model';
 
 const entityComponents = {
   test: testEntityComponents,
+  meModel: meModelComponents,
+  eModel: eModelComponents,
 };
 
 /**
  * Returns an array of extension classes for a particular entity type.
  *
+ * This function will be called in React when we import the extensions module
```

Comment:
Maybe we should put a comment or some jsdoc description for the whole module to make it clear what it does. Also no need to mention React because code here will work with any framework/pure js.